GUI/Utils/DarkModeForms/FlatTabControl.cs

```diff
@@ -0,0 +1,177 @@
+using GUI;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace DarkModeForms
+{
+    public partial class FlatTabControl : TabControl
+    {
+        #region Public Properties
+
+        [Description("Color for a decorative line"), Category("Appearance")]
+        public Color LineColor { get; set; } = MainForm.DarkModeCS.OScolors.Accent;
```

Comment:
`MainForm.DarkModeCS` doesn't exist in designer mode, which is causing it to crash and the tab control to be deleted.